tools/filetree_example.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
@Remi-Gau do you think the changes to this file should be reverted? Or is it fine to keep?